README.md

```diff
@@ -3,6 +3,8 @@
 An application for publishing HyP3 output products for ingest into ASF's data catalog. 
 It contains a HyP3 plugin and an application for publishing jobs sent to it from the HyP3 plugin.
 
+Support is currently limited to publishing the outputs of `S1_ARIA_GUNW` jobs to the `S1_ARIA_GUNW` CMR collection.
```

Comment:
```suggestion
Support is currently limited to publishing the outputs of `ARIA_S1_GUNW` jobs to the `ARIA_S1_GUNW` CMR collection.
```

Reply:
Actually maybe you had the CMR collection right, and it was just the job type that needed to change?